examples/llm_base_model_dequantization/phi_2_dequantization.py

```diff
@@ -0,0 +1,43 @@
+import logging
+
+import yaml
+
+from ludwig.api import LudwigModel
+from ludwig.utils.hf_utils import upload_folder_to_hfhub
+
+base_model_name = "microsoft/phi-2"
+dequantized_path = "microsoft-phi-2-dequantized"
+save_path = "/home/ray/" + dequantized_path
+hfhub_repo_id = "arnavgrg/" + dequantized_path
```

Comment:
Nonblocking: 
Maybe we should use a placeholder instead of "arnavgrg/". Maybe something like:
`{HF_USER}/` + dequantized_path`

Reply:
Otherwise, LGTM.

Reply:
Agreed! Will update